### ISSUE TITLE
incompatible with @types/node@13.13.0

### ISSUE THREAD

**Issue by erkki** [Redocly/redoc#1233]
`yarn run webpack --env.standalone --mode=production` fails with:

```
ERROR in /home/erkki/projects/vw/redoc/node_modules/@types/node/globals.d.ts
ERROR in /home/erkki/projects/vw/redoc/node_modules/@types/node/globals.d.ts(1075,15):
TS2430: Interface 'Require' incorrectly extends interface 'RequireFunction'.
  Types of property 'cache' are incompatible.
    Type 'Dict<NodeModule>' is not assignable to type '{ [id: string]: NodeModule; }'.
      Index signatures are incompatible.
        Type 'NodeModule | undefined' is not assignable to type 'NodeModule'.
          Type 'undefined' is not assignable to type 'NodeModule'.
Child worker:
                                 Asset     Size  Chunks                   Chunk Names
    0807c96575a859b86001.worker.js.map  284 KiB       0  [emitted] [dev]  main
     + 1 hidden asset
    Entrypoint main = 0807c96575a859b86001.worker.js 0807c96575a859b86001.worker.js.map
    [59] (webpack)/buildin/global.js 472 bytes {0} [built]
    [97] ./node_modules/workerize-loader/dist/rpc-worker-loader.js!./node_modules/ts-loader??ref--4-0!./node_modules/babel-loader/lib??ref--4-1!./src/services/SearchWorker.worker.ts + 1 modules 13.8 KiB {0} [built]
         | ./node_modules/workerize-loader/dist/rpc-worker-loader.js!./node_modules/ts-loader??ref--4-0!./node_modules/babel-loader/lib??ref--4-1!./src/services/SearchWorker.worker.ts 3.91 KiB [built]
         |     + 1 hidden module
        + 96 hidden modules
error Command failed with exit code 2.
```

`yarn add @types/node@12.12.6` fixes it for me. Previously on a fresh install `13.13.0` got installed (released very recently).

Maybe consider switching from npm to yarn as the entire dependency tree graph could be locked down then (vs npm package-lock.json)

**Comment by TAnas0** [comment]
I have a completely different setup, but I can confirm such bug can be solved by downgrading to the above mentioned version.

For further details: [StackOverflow question](https://stackoverflow.com/questions/59474983/when-i-npm-runbuild-in-vue-js-get-the-error-interface-noderequire-cannot-simul)

**Comment by imanabu** [comment]
Looks like a Node version issue. This happened on a system where I had stuff build with node 13.0.1 and when we moved the build to a system that had 12.x on it, it broke, and I had to force the type to @12.12.6 as you wrote to get it going. Appreciate your post.

**Comment by RomanHotsiy** [comment]
Fixed in `node@13.11.1`, updated in package-lock.json

You can install precise locked dependency tree by using `npm ci`
